language-php.md

```diff
@@ -44,6 +44,21 @@ Yes:
 $variableName
 ```
 
+### Function (Only PHP7)
+The function we write should declare parameter type and return type, also apply to function annotation.
```

Comment:
For the return type, if the function return something. So here we use `should`, I consider it's a optional option?